app/models/fraud_indicator.rb

```diff
@@ -0,0 +1,102 @@
+# == Schema Information
+#
+# Table name: fraud_indicators
+#
+#  id                   :bigint           not null, primary key
+#  active_at            :datetime
+#  indicator_attributes :string           default([]), is an Array
+#  indicator_type       :string
+#  list_table_name      :string
+#  multiplier           :decimal(, )
+#  name                 :string
+#  points               :integer
+#  reference            :string
+#  source_table_name    :string
+#  threshold            :decimal(, )
+#  created_at           :datetime         not null
+#  updated_at           :datetime         not null
+#
+class FraudIndicator < ApplicationRecord
+  validates :indicator_type, presence: true, inclusion: { in: FraudIndicator.instance_methods - ApplicationRecord.instance_methods }
```

Comment:
I'm planning to add tests for some of these validations, because I think having the validations be accurate per test type can save us from mistakes that would ultimately make the code not work later.